ecs-agent/daemonimages/csidriver/Makefile

```diff
@@ -17,6 +17,14 @@ bin/csi-driver:
 test:
 	go test -v -race -tags unit -timeout=60s ./...
 
+# please include this `mockgen` into PATH
```

Comment:
Yes. Also all unit tests in this new dir will be invoked by the `make test` in the root directory `amazon-ecs-agent`.

> Will update-gomock fail if PATH is not updated?
We don't have to run this update command every time unless there are some related changes. 
And yes if PATH is not updated, the update-gomock will fail because 'we' don't know where the binary is.

Reply:
Covered in another thread.